src/components/Directions.js

```diff
@@ -3,9 +3,9 @@ import React from 'react';
 const Directions = (props) => (
```

Comment:
Let's try to use more of the ES6 features so we can get accustomed to them.
Destructure `props` in the params to require only what is needed, i.e. `({ directionsList })` 
That way the variable can be used w/out needing `props.` in front of it.